native/Cargo.toml

```diff
@@ -39,13 +39,13 @@ arrow-buffer = { version = "52.2.0" }
 arrow-data = { version = "52.2.0" }
 arrow-schema = { version = "52.2.0" }
 parquet = { version = "52.2.0", default-features = false, features = ["experimental"] }
-datafusion-common = { git = "https://github.com/apache/datafusion.git", rev = "bddb641" }
-datafusion = { default-features = false, git = "https://github.com/apache/datafusion.git", rev = "bddb641", features = ["unicode_expressions", "crypto_expressions"] }
-datafusion-functions = { git = "https://github.com/apache/datafusion.git", rev = "bddb641", features = ["crypto_expressions"] }
-datafusion-expr = { git = "https://github.com/apache/datafusion.git", rev = "bddb641", default-features = false }
-datafusion-physical-plan = { git = "https://github.com/apache/datafusion.git", rev = "bddb641", default-features = false }
-datafusion-physical-expr-common = { git = "https://github.com/apache/datafusion.git", rev = "bddb641", default-features = false }
-datafusion-physical-expr = { git = "https://github.com/apache/datafusion.git", rev = "bddb641", default-features = false }
+datafusion-common = { git = "https://github.com/andygrove/datafusion.git", rev = "bddb641" }
```

Comment:
I wil update this once https://github.com/apache/datafusion/pull/11889 is merged and the datafusion repo is tagged with 41.0.0-rc1